sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/BeamSqlEnv.java

```diff
@@ -73,11 +106,27 @@ public void registerUdaf(String functionName, Combine.CombineFn combineFn) {
     defaultSchema.add(functionName, new UdafImpl(combineFn));
   }
 
-  public BeamQueryPlanner getPlanner() {
-    return planner;
+  public PTransform<PCollectionTuple, PCollection<Row>> parseQuery(String query)
+      throws SqlParseException, RelConversionException, ValidationException {
```

Comment:
It would be nice to hide the exceptions inside as well.

Reply:
Makes sense, will do

Reply:
Done